SPEC.md

```diff
@@ -1,6 +1,7 @@
-# Code Climate Engine specification
+# Code Climate Engine Specification
+## Version 0.1
```

Comment:
Seems risky / annoying to have this in two places. I vote it only be specified in the `VERSION` file.